hw/drivers/flash/spiflash/src/spiflash.c

```diff
@@ -628,13 +636,23 @@ static inline void spiflash_unlock(struct spiflash_dev *dev)
 static inline void
 spiflash_cs_activate(struct spiflash_dev *dev)
 {
+#if MYNEWT_VAL(BUS_DRIVER_PRESENT)
+    struct bus_spi_node *node = (struct bus_spi_node *)&dev->dev;
+    hal_gpio_write(node->pin_cs, 0);
```

Comment:
this is redundant since `spiflash_cs_activate` is not used (?) with bus driver as CS line is controlled automatically so perhaps it should be ifdef'd-out